package.json

```diff
@@ -32,8 +32,9 @@
     "docs:lint": "remark -- './**/*.md'",
     "docs:fix": "remark --output -- './**/*.md'",
     "babel": "babel src/js -d es5",
-    "prepublish": "not-in-install && npm run docs:api || in-install",
-    "prepush": "npm run lint -- --errors"
+    "prepublish": "not-in-install && run-p docs:api build || in-install",
+    "prepush": "npm run lint -- --errors",
+    "version": "node build/version.js && git add CHANGELOG.md"
```

Comment:
run the changelog in version only if we're not in prerelease, afterwards, add it to git.